src/style.css

```
@@ -0,0 +1,93 @@
+body {
+  margin: 0;
+}
+
+.product-card {
+  margin: auto;
+  width: 200px;
+  height: 403px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 14px 15px;
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 45px;
+}
+
+.product-card__heading {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-bottom: 5px;
+  margin-top: 0;
```

don't repeat margin property
```suggestion
  margin: 0 0 15px;
```

margin: top right bottom left

---

I did not understand about margins. Here i want to set only top and bottom magins, and i do not want to set a left and right margins, how should i write?

---

since your heading doesn't have default margins for left and right side, you can just you here shorthand property, you'll not break anything in this case. Shorthand property is better to use if possible